libavcodec/vvc/dec.c

```diff
@@ -623,6 +665,10 @@ static av_cold int frame_context_init(VVCFrameContext *fc, AVCodecContext *avctx
         fc->DPB[j].frame = av_frame_alloc();
         if (!fc->DPB[j].frame)
             return AVERROR(ENOMEM);
+
+        fc->DPB[j].frame_grain = av_frame_alloc();
```

Comment:
DPB is not a good place for frame_grain. frame_grain is not included in dpb process.

Reply:
I reference it from hevcdec.
```
    for (i = 0; i < FF_ARRAY_ELEMS(s->DPB); i++) {
        ff_hevc_unref_frame(&s->DPB[i], ~0);
        av_frame_free(&s->DPB[i].frame_grain);
    }
```

May you help check where is a better place to put on?